pkg/resource/deploy/step_generator.go

```diff
@@ -56,6 +56,7 @@ func (sg *stepGenerator) GenerateReadSteps(event ReadResourceEvent) ([]Step, err
 		event.Parent(),
 		false, /*protect*/
 		true,  /*external*/
+		"",    /* status */
```

Comment:
Can you use the constants you defined here?

Reply:
yep - I switched from strings to constants halfway through this PR and evidently I missed a few places.